{{cookiecutter.project_slug}}/.github/workflows/expo-emergency-prod-update.yml

```diff
@@ -0,0 +1,38 @@
+name: Expo bugfix update - used to quickly push a fix to prod without a full rebuild
+on:
+  workflow_dispatch:
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v2
+
+      - uses: actions/cache@v2
+        with:
+          path: "**/node_modules"
+          key: {{ "${{ runner.os }}-modules-${{ hashFiles('**/yarn.lock') }}" }}
+
+      - name: 🏗 Setup Node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - name: 🏗 Setup Expo and EAS
+        uses: expo/expo-github-action@v7
+        with:
+          expo-version: latest
+          eas-version: latest
+          token: {{ "${{ secrets.EXPO_TOKEN }}" }}
```

Comment:
The extra pair of curliest is both confusing and annoying, glad you were able to figure it out on your own 😄

Reply:
Happened to me as well on a React file where we had a prop such as `style={ { fontSize: 12 } }` and it was blowing up as well, so we had to do something similar than here